src/main/java/nextstep/favorite/application/FavoriteService.java

```diff
@@ -0,0 +1,35 @@
+package nextstep.favorite.application;
+
+import nextstep.favorite.application.dto.FavoriteRequest;
+import nextstep.favorite.application.dto.FavoriteResponse;
+import nextstep.favorite.domain.Favorite;
+import nextstep.favorite.domain.FavoriteRepository;
+import nextstep.subway.applicaion.StationService;
+import nextstep.subway.domain.Station;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class FavoriteService {
+
+    private FavoriteRepository favoriteRepository;
+    private StationService stationService;
+
+    public FavoriteService(FavoriteRepository favoriteRepository, StationService stationService) {
+        this.favoriteRepository = favoriteRepository;
+        this.stationService = stationService;
+    }
+
+    public void addFavorite(FavoriteRequest favoriteRequest) {
+        Station source = stationService.findById(favoriteRequest.getSourceId());
+        Station target = stationService.findById(favoriteRequest.getTargetId());
+        Favorite favorite = new Favorite(favoriteRequest.getMemberId(), source, target);
+        favoriteRepository.save(favorite);
+    }
+
+    public List<FavoriteResponse> findFavorite(FavoriteRequest favoriteRequest) {
+        List<Favorite> favoriteList = favoriteRepository.findAllByMemberId(favoriteRequest.getMemberId());
+        return favoriteList.stream().map(FavoriteResponse::of).collect(Collectors.toList());
```

Comment:
Stream API를 사용할 때 각 단계 별로 개행을 해주면 pipeline을 파악하기가 간결 편해집니다 :)

```suggestion
        return favoriteList.stream()
                .map(FavoriteResponse::of)
                .collect(Collectors.toList());
```